pkg/controller/backup_controller.go

```diff
@@ -349,6 +349,13 @@ func (controller *backupController) getLocationAndValidate(itm *api.Backup, defa
 		itm.Spec.StorageLocation = defaultBackupLocation
 	}
 
+	// add the storage location as a label for easy filtering later.
```

Comment:
Should we also add this when creating the backup?

Reply:
in the CLI code you mean? Wouldn't hurt, though everything should end up going thru this code path anyway

Reply:
I actually meant the controller. It seems a little disjointed to set information for a new backup in both the backup controller and sync controller, to me. I do think this should stay for backups imported from a newly added location, though.

Reply:
This is the backup controller

Reply:
Maybe I'm confused about your comment.

Reply:
:man_facepalming: nevermind then :)

Reply:
Ah, you mean additionally do this in the backup sync controller when importing? Yeah, that's probably necessary.  I'm thinking we are also going to need to rewrite the `StorageLocation` field since the name of it in the importing cluster may be different than the name in the source cluster.

Reply:
> I'm thinking we are also going to need to rewrite the StorageLocation field since the name of it in the importing cluster may be different than the name in the source cluster.

Right, that's what I thought this was doing, but thought I was reading the sync controller.